### ISSUE TITLE
Issues when providing options in sveltepreprocessor

### ISSUE THREAD

**Issue by navneetsharmaui** [mihar-22/svelte-jester#38]
Jest is not performing the tests when sveltepreprocessor contains the config values

```javascript
	preprocess: [
		sveltePreprocess({
			defaults: {
				style: 'postcss',
			},
			postcss: true,
		}),
	],
```

And If those values are removed it performs the tests but the test fails because most of the components contain the postcss styles.

![Screenshot from 2021-04-16 19-30-23](https://user-images.githubusercontent.com/11630812/115035522-490c2280-9eea-11eb-8cc4-80931825059f.png)

The Header and Card components contain the postcss styles in them, and it's just displays pending for them.

**Comment by sebastianrothe** [comment]
Hm, I also have those values and my tests are run. What are you using exactly? Tailwind with PostCSS? JIT-Mode?

**Comment by navneetsharmaui** [comment]
@sebastianrothe here is the repo https://github.com/navneetsharmaui/sveltekit-starter/tree/jest-unit-test-setup

**Comment by sebastianrothe** [comment]
The problem is the jit-mode. Jest hangs, while the jit-watcher runs in the background. You need to pass `TAILWIND_MODE=build` when running the tests. So `cross-env TAILWIND_MODE=build jest` in your `package.json` for example. 

Duplicate of #39. Should be fixed in the latest version.

**Comment by navneetsharmaui** [comment]
@sebastianrothe Yes you are correct the issue was related to the jit-mode of tailwindcss. The test seems to work the fix provided in the #40
